ui/plugin-lifecycle/src/main/classes/XMLHelper.ts

```diff
@@ -0,0 +1,39 @@
+export abstract class XMLHelper {
```

Comment:
Where do you still need XML processing?  Are there API calls that don't work with JSON?

Reply:
It is not used at the moment, I'm removing it in last commit.